File: 1_year/algoritmos/first day/30_medias.py
i = 0

while i < 3:
    nota1 = int(input("Insira a nota 1 "))
    nota2 = int(input("Insira a nota 2 "))

    media = (nota1 + nota2) / 2

    if media >= 7:
        print("Aprovado")
    else:
        print("Reprovado")

    i += 1

File: 1_year/algoritmos/first day/30_medias.c
#include <stdio.h>

int main()
{

    int i = 0;

    while (i < 3)

    {
        int nota1 = 0;
        int nota2 = 0;
        printf("\n Insira a nota 1: ");
        scanf("%d", &nota1);
        printf("Insira a nota 2: ");
        scanf("%d", &nota2);

        int media = (nota1 + nota2) / 2;

        if (media < 7)
        {
            printf("Reprovado");
        }
        else
        {
            printf("Aprovado");
        };

        i += 1;
    };

    return 0;
};
